backend/src/routes/route.js

```diff
@@ -1,11 +1,13 @@
-const express=require('express')
-const upload=require('../middleware/multer')
-const processController= require('../controllers/process')
-const router=new express.Router();
+const express = require('express')
+const upload = require('../middleware/multer')
+const processController = require('../controllers/process')
+const saveDetails = require('../controllers/dummyapi');
```

Comment:
Dont require controller twice
create an index file in controller folder, export all controller files through index file.
for require either import whole controller or destruct controller object

Reply:
done